pixel-saver@deadalnix.me/decoration.js

```diff
@@ -109,7 +109,9 @@ function guessWindowXID(win) {
 
 	// debugging for when people find bugs..
 	WARN("Could not find XID for window with title %s".format(win.title));
```

Comment:
This WARN call should better be changed to LOG. It's perfectly fine if no original state could be found. No reason to pollute the message log by default.